R/build-news.R

```diff
@@ -249,13 +249,13 @@ tweak_news_heading <- function(x, versions, timeline) {
   date <- timeline$date[match(versions, timeline$version)]
   date_str <- ifelse(is.na(date), "Unreleased", as.character(date))
 
-  date_nodes <- paste(" <small>", date_str, "</small>", collapse = "") %>%
+  date_nodes <- paste(" <h6 data-toc-skip> CRAN release: ", date_str, "</h6>", collapse = "") %>%
```

Comment:
I'd prefer to keep the existing style — we can use `<h2 data-toc-text="Short text">Longer text</h2>` to customise what appears in the TOC